src/app/static/src/tests/filteredData/getters.test.ts

```diff
@@ -500,13 +500,13 @@ describe("FilteredData mutations", () => {
                 iso3: "MWI",
                 area_id: "area1",
                 ancrt_test_pos: 2,
-                age_group_id: "1"
+                age_group_id: 1
```

Comment:
We're deriving our types from the json schema - I think the id here must have changed from string to number? I don't think it really matters but it would be better if they were consistent of course. Are they all numbers now?

Reply:
I'm not fully sure - I can see a possibility where user upload data with this as a character representation of the number. It depends on the source though, I will just create a ticket to make sure that we're converting to number if it is a string.